testing-bootstrap/testing-bootstrap.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+  id "com.github.johnrengelman.shadow"
+}
+
+apply from: "$rootDir/gradle/java.gradle"
+
+// Depend on all libraries that are in the bootstrap classloader when running the agent. When
+// running tests, we simulate this by adding the jar produced by this project to the bootstrap
+// classpath.
+
+dependencies {
+  implementation project(":instrumentation-api")
+  implementation project(":javaagent-bootstrap")
+  implementation project(path: ":opentelemetry-sdk-shaded-for-testing", configuration: "shadow")
```

Comment:
@trask Interestingly none of the instrumentation tests failed with this off the bootstrap, but `AgentTestRunner` which verifies the test classpath explicitly did. Do you know if anything we've done here has removed the requirement for this to be on the bootstrap?

Reply:
I don't think it needs to be on the bootstrap, could have been an earlier confusion of mine, check out commit I pushed to this PR below